.github/workflows/chromatic.yml

```diff
@@ -4,12 +4,15 @@
 name: 📖 Storybook (via Chromatic)
 
 # Event for the workflow
-on: push
+on:
```

Comment:
We might also keep doing it for the push against upstream main
https://github.com/atlasmap/atlasmap/blob/main/.github/workflows/supported-build.yml#L7-L8